app/modules/entities/components/editor/claim_editor.svelte

```diff
@@ -20,9 +20,9 @@
   const dispatch = createEventDispatcher()
   const { uri, type } = entity
   const creationMode = uri == null
-  const { editorType, canValueBeDeleted, specialEditActions } = propertiesEditorsConfigs[property]
-  const { InputComponent, DisplayComponent, showSave } = editors[editorType]
-  const fixed = editorType.split('-')[0] === 'fixed'
+  const { datatype, canValueBeDeleted, specialEditActions } = propertiesEditorsConfigs[property]
```

Comment:
```suggestion
  const { valuetype, canValueBeDeleted, specialEditActions } = propertiesEditorsConfigs[property]
```
Unless some types are not property values per se, valuetype feel more precise and fit with wikidata naming.

[non-blocking]

Reply:
`datatype` is the attribute used by Wikidata to categorize properties by snak value type  (ex: [`P123`](https://www.wikidata.org/w/api.php?action=wbgetentities&format=json&ids=P123&props=info)). We differ in some cases in the possible `datatype` values (ex: `entity` rather than `wikibase-item`), but I think the homonymy is useful.